### DISCUSSION TITLE
Background texture

### DISCUSSION THREAD

**Discussion by bastado** [zsviczian/obsidian-excalidraw-plugin#203] [Q&A] [answered]
Hi is it possible to change the background. That would be veryhelpfull for Math for example.

**Comment by YesterKo** [comment, marked as answer]
What do you want to change it to?
If you want a grid, you can enable it by right-clicking and selecting `show grid`. If you need to change the size of the grid, you can change it in the markdown view, under `"gridSize":`

**Comment by bastado** [comment]
Thanks for your answer it was very helpfull.
Von: YesterKo
Gesendet: Mittwoch, 27. Oktober 2021 12:59
An: zsviczian/obsidian-excalidraw-plugin
Cc: obsidiansketcher; Author
Betreff: Re: [zsviczian/obsidian-excalidraw-plugin] Background texture(Discussion #203)

What do you want to change it to?
If you want a grid, you can enable it by right-clicking and selecting show grid. If you need to change the size of the grid, you can change it in the markdown view, under "gridSize":
—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Reply by YesterKo** [reply]
Glad to help. If it solves your problem, you can mark it as the answer and close the question :)
